models/comments.model.js

```diff
@@ -0,0 +1,33 @@
+const db = require('../db/connection');
+
+exports.fetchComments = (articleId) => {
+    return db.query(`
+        SELECT * 
+        FROM comments 
+        WHERE article_id = $1
+        ORDER BY created_at DESC;
+    `, [articleId])
+    .then((data) => {
+        return data.rows;
+    })
+    .catch((err) => {
+        return err;
+    })
+}
+
```

Comment:
doesn't your getArticleById function already check for the existence of an article?

Reply:
oh, i misunderstood you. i guess getArticleById returns an error, but Paul showed us in the morning this approach of independent checking of existence and i decided to try it.